src/dust/dust.cpp

```diff
@@ -207,6 +257,14 @@ std::shared_ptr<StateDescriptor> Initialize(ParameterInput *pin) {
   m.SetSparseThresholds(0.0, 0.0, 0.0);
   dust->AddSparsePool<dust::prim::velocity>(m, control_field, dustids);
 
+  // dust stopping-time
+  m = Metadata({Metadata::Cell, Metadata::Derived, Metadata::Intensive, Metadata::OneCopy,
+                Metadata::Sparse});
+  m.SetSparseThresholds(0.0, 0.0, 0.0);
+  dust->AddSparsePool<dust::stopping_time>(m, control_field, dustids);
```

Comment:
Can be removed

Reply:
see my comment above

Reply:
removed in the new updated code and calculate the stopping time on fly